test/TestProjects/OperationGroupMappings/OperationGroupMappings.json

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
for the other testprojects this file is in the generated folder.

Reply:
Looks like they have it not in the generated folders in test/testProjects

![image](https://user-images.githubusercontent.com/53361486/108135346-d0fdb980-706c-11eb-8210-203626cf7193.png)